app/src/main/res/layout/fragment_advanced_reminder_settings.xml

```diff
@@ -15,6 +15,8 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_margin="4dp"
+            android:focusable="true"
```

Comment:
Do you still need this change after the code change posting the listener?

Reply:
Yes, both changes are necessary.
- The change in f111b18ad0be1bc93c8fcbeffff23981481ee89d prevents the date picker from being loaded when entering the advanced reminder settings.
- The change in f078ef3eb2918ce9dcacb41bff95e14800c7cfcb ensures the date picker does not load accidentally after exiting the advanced reminder settings.